ml-agents/mlagents/trainers/tests/torch/test_utils.py

```diff
@@ -50,7 +51,7 @@ def test_create_inputs(encoder_type, normalize, num_vector, num_visual):
         obs_shapes.append(vis_obs_shape)
     h_size = 128
     sen_spec = create_sensor_specs_with_shapes(obs_shapes)
-    encoders, embedding_sizes = ModelUtils.create_input_processors(
+    encoders, embedding_sizes, _ = ModelUtils.create_input_processors(
```

Comment:
Do we think EntityEmbeddings should live in torch/utils.py since its an input processor?